emsdk.py

```diff
@@ -2842,6 +2842,18 @@ def expand_sdk_name(name, activating):
 
 
 def main(args):
+
+  if is_emsdk_sourced_from_github():
+    # This code only exists on the master branch
+    errlog('****')
+    errlog('Warning: You appear to be using the `master` branch of emsdk.')
+    errlog('We recently made the switch to using `main`')
+    errlog('In order to continue to receive updates will need to make the switch locally too.')
+    errlog('For trivial checkouts simply running the following command should be enough:')
+    errlog('  `git checkout origin/main`')
```

Comment:
why `origin/main` instead of `main`?

Reply:
`main` does not exist after a fetch.. only `origin/main` exists at that point..

I guess `git checkout` does automatically create local branches based on remote ones do `git check main` will technically work.   Maybe I'm biased because I tend to always qualify my banch names when I want to refer to a remote branch like this.

Reply:
Yes, it will automatically create a local branch for it, that's why I think it's ok to do. It's what I always depend on myself.

Plain `main` seems closer to plain `master`. Also it will be closer to what a user gets on a fresh clone. That's why I think it's simpler.

Reply:
Yeah, I think you are right.   

I'm an anomaly because I never have local branch called `master` or `main` since I use `git rebase-update` all the time and it (quite rightly I guess) deletes any branch that are the same as their upstream branch (i.e. it assumed they have been merged).    Local `main` has no point to exist in this workflow as `origin/main` is the upstream target of any local branches.